src/components/utils/utils.js

```diff
@@ -98,3 +98,46 @@ export const buildNewBusbarSections = (
     }
     return newBusbarSections;
 };
+
+export function calculateResistance(distance, linearResistance) {
+    if (
+        distance === undefined ||
+        isNaN(distance) ||
+        linearResistance === undefined ||
+        isNaN(linearResistance)
+    ) {
+        return 0;
+    }
+    return Number(distance) * Number(linearResistance);
+}
+
+export function calculateReactance(distance, linearReactance) {
+    if (
+        distance === undefined ||
+        isNaN(distance) ||
+        linearReactance === undefined ||
+        isNaN(linearReactance)
+    ) {
+        return 0;
+    }
+    return Number(distance) * Number(linearReactance);
+}
```

Comment:
They do the same thing, but they have different roles. I am not a fan of either of those options :

- Use only one function called "calculateReactanceOrResistance" => What's the next step ? "calculateReactanceOrResistanceOrRadiusOrWhateverThatsJustMathAnyway" ? :)
- Do not use a function and directly do the math => With specific functions, we can use and maintain them easily.

Do you have another better solution ?

Reply:
I am wondering if this wouldn't do the same thing, and would make it a one line operation instead of duplicate functions :
 
distance * linearReactance || 0

if one of them is not a number or is undefined, it will return NaN, and NaN || 0 = 0